### ISSUE TITLE
Cache Docker images in the daily accessibility scan

### ISSUE THREAD

**Issue by ddzz** [department-of-veterans-affairs/va.gov-team#31818]
The daily accessibility scan is failing occasionally due to Docker pull rate limit errors for the Docker image used in Slack notifications. This task is to cache this Docker image in the GitHub Actions workflow so that the rate limit is not exceeded.

**Comment by ddzz** [comment]
This PR circumvents the issue: https://github.com/department-of-veterans-affairs/content-build/pull/727
